pkg/apis/storage/types.go

```diff
@@ -672,3 +672,43 @@ type CSIStorageCapacityList struct {
 	// Items is the list of CSIStorageCapacity objects.
 	Items []CSIStorageCapacity
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// VolumeAttributesClass represents a class of volume attributes. It holds the mutable
+// attributes of volumes for the provisioner that should create and update volumes.
+//
+// VolumeAttributesClasses are non-namespaced; the name of the volume attributes class
+// according to etcd is in ObjectMeta.Name.
+type VolumeAttributesClass struct {
+	metav1.TypeMeta
+
+	// Standard object's metadata.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+	// +optional
+	metav1.ObjectMeta
+
+	// parameters holds the mutable attributes of volumes for the provisioner that should
+	// create and update volumes of this volume attributes class. And these parameters are
+	// immutable.
+	// +optional
+	Parameters map[string]string
+}
```

Comment:
@sunnylovestiramisu Has the `DriverName string` been dropped from VAC?  It was in the KEP.

Reply:
The driver name should not be dropped. Please add it.

Reply:
@sunnylovestiramisu It was added in https://github.com/kubernetes/kubernetes/pull/118863/commits/ab26c855cc7975be51c3bdf77a46202fa64dbdac